README.md

```diff
@@ -63,6 +63,12 @@ The **Budget app** keeps track of your transcations It allows the user to:
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+### Live demo <a name="live-demo"></a>
+
+Click this [live demo](https://budget-app-0yss.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] [REQUIRED] It looks like the deployment was not successful. Let's take a look at the issues then kindly redeploy as stated by the project requirement 💪🏽